test/SlowTests/Sharding/Cluster/ReshardingTests.cs

```diff
@@ -544,7 +544,7 @@ public async Task BucketDeletionShouldCreateArtificialTombstones2()
                 using (context.OpenReadTransaction())
                 {
                     var tombs = db.DocumentsStorage.GetTombstonesFrom(context, 0).ToList();
-                    Assert.Equal(20, tombs.Count);
+                    Assert.Equal(22, tombs.Count);
```

Comment:
why do we have more tombstones now?

Reply:
we are creating timeseries on 2 documents now so we have 2 more revisions in bucket, and after migration we have 2 more revision tombstones